app/labor/markdown_fixer.rb

```diff
@@ -54,10 +54,10 @@ def underscores_in_usernames(markdown)
 
     def add_quotes_to_section(markdown, section:)
       # Only add quotes to front matter, or text between triple dashes
-      markdown.gsub(FRONT_MATTER_DETECTOR) do |front_matter|
-        front_matter.gsub(/#{section}:\s?(.*?)(\r\n|\n)/m) do |target|
-          # 1 is the captured group (.*?)
-          captured_text = Regexp.last_match(1)
+      markdown.sub(FRONT_MATTER_DETECTOR) do |front_matter|
```

Comment:
Changing `gsub` to `sub` is a nice touch.

Reply:
B)